src/vaadin-grid-sort-mixin.js

```diff
@@ -51,31 +51,66 @@ export const SortMixin = (superClass) =>
     /** @private */
     _onSorterChanged(e) {
       const sorter = e.target;
+      e.stopPropagation();
+      if (this._isUpdatingSorter) return;
+      try {
+        this._isUpdatingSorter = true;
+        this.__updateSorter(sorter);
+        this.__applySorters();
+      } finally {
+        this._isUpdatingSorter = false;
+      }
+    }
+
+    /** @private */
+    __removeSorters(sorters) {
+      if (sorters.length == 0) {
+        return;
+      }
+
+      sorters.forEach((sorter) => {
+        this._removeArrayItem(this._sorters, sorter);
+        if (this.multiSort) {
+          this.__updateSortOrders();
```

Comment:
Can we update `sortOrders` once all sorters are removed? Not to call it after each sorter is removed.

Reply:
Done